src/System.Net.Mail/src/System/Net/Mail/SmtpClient.cs

```diff
@@ -748,6 +749,18 @@ public void SendAsync(MailMessage message, object userToken)
             }
         }
 
+        private bool IsSNCInCredentialCache(CredentialCache cache)
```

Comment:
Nit: I know it's long, but can you spell out the acronym, e.g. IsSystemNeworkCredentialInCache?  Just looking at the call site, I had no idea what SNC was.